app/controllers/properties_controller.rb

```diff
@@ -9,7 +9,11 @@ def index
                           .per(Settings.account.per_page)
   end
 
-  def show; end
+  def show
+    @agent = @property.account
+    @agent_properties = Property.where(account_id: @agent.id)
```

Comment:
nên dùng association sẵn có nha
```suggestion
    @agent_properties = @agent.properties
```